README.md

```diff
@@ -246,17 +257,23 @@ Before continuing, check the following:
 
 To deploy:
 
-1. Go to [repository actions](/../../actions/workflows/web_app.yml) and click on _Run workflow_ and then the green button to run it.
+1. Navigate to your repository on Github and select the _Actions_ tab.
+2. Select _Build and deploy Python app to Azure Web App_ from the _Workflows_ sidebar.
+3. Select _Run workflow_ and select the green button inside the pop-up to run the workflow.
```

Comment:
Same reasoning here as above. Also, the green button is inside a pop-up, so I clarified that. There's also a weird pause between clicking the button and anything happening, hopefully Github is improving that UI.

Reply:
That's right, there is always a pause with GitHub Actions and thanks for clarifying this, it reads better